test/integration/clusterio.js

```diff
@@ -32,6 +57,35 @@ describe("Integration of Clusterio", function() {
 				);
 			});
 		});
+		describe("Live updating", function() {
+			let slaveProcess;
+			it("Slaves", async function() {
+				slowTest(this);
+				let testPromise = new Promise((resolve, reject) => {
+					let control = getControl();
+					control.onLiveSlaveAdded("testing", async function() {
+						resolve();
+					});
+				});
+				if (await fs.exists("./temp/test/liveUpdates/slave-config.json")) {
+					await fs.unlink("./temp/test/liveUpdates/slave-config.json");
+				}
+				/* eslint-disable max-len */
+				await execCtl("slave create-config --id 5 --name slave2 --generate-token --output ./liveUpdates/slave-config.json");
+				await exec("node ../../packages/slave config set slave.tls_ca ../../test/file/tls/cert.pem --config ./liveUpdates/slave-config.json");
+				slaveProcess = await spawn("slave2:", "node ../../packages/slave run --config ./liveUpdates/slave-config.json", /SOCKET \| registering slave/);
+				/* eslint-enable max-len */
```

Comment:
The max line length rule is there for a reason. Break up these lines into shorter ones.